content/posts/0022-bionic-eos.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Bionic End of Support
+date: "2023-07-28"
+slug: bionic-eos
+author: swigmore
+---
+
+As of May 31, 2023, Ubuntu 18.04 (Bionic Beaver) is [out of
+support](https://ubuntu.com/blog/18-04-end-of-standard-support) for open-source
+customers.
+
+As a result, Paketo will stop supporting Bionic in the project. If you
+currently depend on our Bionic-based offerings, this blog post will contain
+all of the relevant information about what this means and how to migrate.
+
+## Deprecation Details
```

Comment:
Nit: s/Deprecation/End of Support/? (no strong opinion)

Reply:
Given this times timing the thing happening is an EOL. However, if we want to set precedent for the future, we should announce the deprecation of Jammy way before we stop publishing updates...

So, I agree that both would work here.

Reply:
Ill change it to End of Support!